### PR TITLE
Greek translation

### PR DESCRIPTION
**Pull request by GreekGeorge**
I did the yarn thing. Here's the greek translation files.

### CODE REVIEW

**Comment by yanfali** [comment]
@yiancar please review this one

File: src/i18n/gr/index.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+      },
+      layout: {
+        label: 'διάταξη'
+      },
+      keymapName: {
+        label: 'Όνομα Διάταξης',
+        placeholder: 'προσαρμοσμένο όνομα keymap'
+      },
+      loadDefault: {
+        label: 'Φότωση Προεπιλεγμένων',
```
Comment (yiancar) [inline src/i18n/gr/index.js]: Φόρτωση ;

**Comment by yiancar** [comment]
Just one small fix from me:)

File: src/i18n/gr/index.js (hunk 1 above)
Comment (GreekGeorge) [reply src/i18n/gr/index.js]: Yes it should be φόρτωση, I must have missed it. Good catch yiancar!

**Comment by yanfali** [comment]
@GreekGeorge let me know when you are done and I will be happy to merge.

**Comment by yiancar** [comment]
Up to now, looks good to me!

**Comment by GreekGeorge** [comment]
@yiancar I updated the edit on my repository. It should be good now.

**Pull request merged**